lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -75,6 +75,20 @@ module Resources
                 'start': '2019-12-26T16:06:26.000Z'
               },
               'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'cod': '152',
```

Comment:
'cod' -> 'code'

Reply:
Applies through the examples.

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR148 and through out the examples

Reply:
+1 to the 'cod' -> 'code' changes.